src/plugins/kibana_react/public/code_editor/languages/types.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { monaco } from '@kbn/monaco';
+
+export interface LangModule {
```

Comment:
I'm not sure about having that interface in that place cause in @kbn/monaco we work with the same object

![image](https://user-images.githubusercontent.com/20072247/120607796-32e60180-c459-11eb-9b3b-37b78ad2af13.png)

What do you think about creating that interface in @kbn/monaco  and re-export it?

Reply:
Interface, which is used in @kbn/monaco is providing its own interface, different from interface, defined in monaco package. They are using conf, language and ID. I wished to match their interface to avoid confusing of users (programmers, in our case). But if what, I can change it.